pyisic/_standards/nacebel2008/nacebel2008.py

```diff
@@ -0,0 +1,3976 @@
+from ...types import Classification, Standard, Standards
+
+NACEBEL2008 = Standard(
```

Comment:
Take a look at the [NACE2 standard](https://github.com/sayari-analytics/pyisic/blob/main/pyisic/_standards/nace2/nace2.py) I think some of these qualify as different categories e.g. Section vs Class)